File: ios/Classes/message/entity/video/VideoSnapshotInfo.swift
//
//  VideoSnapshotInfo.swift
//  tencent_im_plugin
//
//  Created by 蒋具宏 on 2020/3/15.
//  视频缩略图信息
public class VideoSnapshotInfo : NSObject{
    /// 图片ID
    var uuid : String?;
    
    /// 宽度
    var width : Int32?;
    
    /// 高度
    var height : Int32?;
    
    /// 大小
    var size : Int32?;
    
    /// 类型
    var type : String?;
    
    /// 路径
    var path : String?;
}

File: android/src/main/java/top/huic/tencent_im_plugin/message/entity/video/VideoSnapshotInfo.java
package top.huic.tencent_im_plugin.message.entity.video;

import java.io.Serializable;

/**
 * 视频缩略图信息
 *
 * @author 蒋具宏
 */
public class VideoSnapshotInfo implements Serializable {
    /**
     * 图片ID
     */
    private String uuid;

    /**
     * 图片大小
     */
    private Long size;

    /**
     * 图片宽度
     */
    private Long width;

    /**
     * 图片类型
     */
    private String type;

    /**
     * 图片高度
     */
    private Long height;

    /**
     * 路径
     */
    private String path;

    public Long getSize() {
        return size;
    }

    public void setSize(Long size) {
        this.size = size;
    }

    public Long getWidth() {
        return width;
    }

    public void setWidth(Long width) {
        this.width = width;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Long getHeight() {
        return height;
    }

    public void setHeight(Long height) {
        this.height = height;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }
}